seqr/views/utils/export_utils_2_3_tests.py

```diff
@@ -14,23 +16,23 @@ class ExportTableUtilsTest(TestCase):
 
     def test_export_table(self):
         header = ['column1', 'column2']
-        rows = [['row1_v1', 'row1_v2'], ['row2_v1', 'row2_v2']]
+        rows = [['row1_v1\xe2', 'row1_v2'], ['row2_v1', 'row2_v2']]
 
         # test tsv format
         response = export_table('test_file', header, rows, file_format='tsv')
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.get('content-disposition'), 'attachment; filename="test_file.tsv"')
-        self.assertEqual(response.content, '\n'.join(['\t'.join(row) for row in [header]+rows]) + '\n')
+        self.assertEqual(response.content, 'column1\tcolumn2\nrow1_v1\xe2\trow1_v2\nrow2_v1\trow2_v2\n'.encode('utf-8'))
```

Comment:
why does this need to change, it looks like it should evaluate to the same thing?

Reply:
The main purpose of the change is to add `.encode('utf-8')` which is required by Python 3 since `response.content` is a binary.

Reply:
then why not leave it as it is and just add the `encode`?

Reply:
I thought the unicodes in the data should be removed. I would like to strip the unicodes in the expected data manually and explicitly rather than using `ascii` encoding. But, later I found the unicode is still there and simply added the char back without changing the whole statement back.
I can change it back.